applications/mne_analyze/mne_analyze/mainwindow.cpp

```diff
@@ -419,9 +420,11 @@ void MainWindow::createPluginControls(QSharedPointer<ANSHAREDLIB::PluginManager>
 void MainWindow::createPluginViews(QSharedPointer<PluginManager> pPluginManager)
 {
     m_pGridLayout = new QGridLayout(this);
-    m_pMultiView = new MultiView();
+    m_pMultiView = new MultiView(m_sSettingsPath);
+    m_pMultiView->setParent(this);
```

Comment:
is this needed? Doesn't the setCentralWidget call below already takes care of parenting the new multi view widget